server/app/controllers/dev/seeding/QuestionDefinitions.java

```diff
@@ -0,0 +1,208 @@
+package controllers.dev.seeding;
+
+import com.google.common.collect.ImmutableList;
+import services.LocalizedStrings;
+import services.question.QuestionOption;
+import services.question.types.AddressQuestionDefinition;
+import services.question.types.CurrencyQuestionDefinition;
+import services.question.types.DateQuestionDefinition;
+import services.question.types.EmailQuestionDefinition;
+import services.question.types.EnumeratorQuestionDefinition;
+import services.question.types.FileUploadQuestionDefinition;
+import services.question.types.IdQuestionDefinition;
+import services.question.types.MultiOptionQuestionDefinition;
+import services.question.types.MultiOptionQuestionDefinition.MultiOptionQuestionType;
+import services.question.types.NumberQuestionDefinition;
+import services.question.types.PhoneQuestionDefinition;
+import services.question.types.QuestionDefinitionConfig;
+import services.question.types.StaticContentQuestionDefinition;
+import services.question.types.TextQuestionDefinition;
+
+public final class QuestionDefinitions {
+
+  public static final AddressQuestionDefinition ADDRESS_QUESTION_DEFINITION =
+      new AddressQuestionDefinition(
+          QuestionDefinitionConfig.builder()
+              .setName("Sample Address Question")
+              .setDescription("description")
+              .setQuestionText(LocalizedStrings.withDefaultValue("What is your address?"))
+              .setQuestionHelpText(LocalizedStrings.withDefaultValue("help text"))
+              .build());
```

Comment:
Nit: add blank space after question definition for consistency. Same on line 39.

Reply:
Done.